server/paymentProviders/transferwise/index.ts

```diff
@@ -138,6 +208,99 @@ async function payExpense(
   return { quote, recipient, transfer, fund, paymentOption };
 }
 
+const makeBatchName = (expenses: Array<typeof models.Expense>) => {
+  const hash = crypto.createHash('sha1');
+  expenses.map(e => e.id).forEach(id => hash.update(id.toString()));
+  return hash.digest('hex');
+};
+
+async function createExpensesBatchGroup(
+  host: typeof models.Collective,
+  expenses: Array<typeof models.Expense>,
+): Promise<BatchGroup> {
+  expenses.forEach(e => {
+    if (!e.PayoutMethod) {
+      throw new BadRequest('payExpenseBatch: Expense is missing PayoutMethod', undefined, { expenseId: e.id });
+    }
+  });
+
+  const [connectedAccount] = await host.getConnectedAccounts({
+    where: { service: 'transferwise', deletedAt: null },
```

Comment:
Do we really need `{ deletedAt: null }`? Isn't this the default in Sequelize?

Reply:
True. 🎯